synapse/models/economic.py

```diff
@@ -296,5 +339,125 @@ def getModelDefs(self):
                     ('price:high', ('econ:price', {}), {
                         'doc': 'The high price of the security'}),
                 )),
+
+                ('econ:acct:invoice', {}, (
+
+                    ('issued', ('time', {}), {
+                        'doc': 'The time that the invoice was issued to the recipient.'}),
+
+                    ('issuer', ('ps:contact', {}), {
+                        'doc': 'The contact information for the entity who issued the invoice.'}),
+
+                    ('recipient', ('ps:contact', {}), {
+                        'doc': 'The contact information for the intended recipient of the invoice.'}),
+
+                    ('paid', ('bool', {}), {
+                        'doc': 'Set to true if the invoice has been paid in full.'}),
+
+                    ('balance', ('econ:price', {}), {
+                        'doc': 'The balance due.'}),
+
+                    ('currency', ('econ:currency', {}), {
+                        'doc': 'The currency that the invoice specifies for payment.'}),
+                )),
+
+                ('econ:acct:receipt', {}, (
+
+                    ('issued', ('time', {}), {
+                        'doc': 'The time the receipt was issued.'}),
+
+                    ('purchase', ('econ:purchase', {}), {
+                        'doc': 'The purchase that the receipt confirms payment for.'}),
+
+                    ('issuer', ('ps:contact', {}), {
+                        'doc': 'The contact information for the entity who issued the receipt.'}),
+
+                    ('recipient', ('ps:contact', {}), {
+                        'doc': 'The contact information for the entity who received the receipt.'}),
+
+                    ('currency', ('econ:currency', {}), {
+                        'doc': 'The currency that the receipt uses to specify the price.'}),
+
+                    ('amount', ('econ:price', {}), {
+                        'doc': 'The price that the receipt confirms was paid.'}),
+                )),
+
+                ('econ:bank:aba:rtn', {}, (
+
+                    ('bank', ('ou:org', {}}), {
+                        'doc': 'The bank which was issued the ABA RTN.'}),
+
+                    ('bank:name', ('ou:name', {}), {
+                        'doc': 'The name which is registered for this ABA RTN.'}),
+
+                )),
+
+                ('econ:bank:iban', {}, ()),
+
+                ('econ:bank:swift:bic', {}, (
+
+                    ('business', ('ou:org', {}), {
+                        'doc': 'The business which is the registered owner of the SWIFT BIC.'}),
+
+                    ('office', ('ps:contact', {}), {
+                        'doc': 'The branch or office which is specified in the last 3 digits of the SWIFT BIC.'}),
+                )),
+
+                ('econ:bank:account:type:taxonomy', {}, ()),
+                ('econ:bank:account', {}, (
```

Comment:
Is payment card currently linked to an account? Does the econ:pay:card form need to have an econ:bank:account property type added?